pkgs/development/python-modules/pwntools/default.nix

```diff
@@ -1,8 +1,8 @@
 { stdenv, buildPythonPackage, fetchPypi, isPy3k
 , Mako, packaging, pysocks, pygments, ROPGadget
 , capstone, paramiko, pip, psutil
-, pyelftools, pypandoc, pyserial, dateutil
-, requests, tox, pandoc, unicorn, intervaltree }:
+, pyelftools, pyserial, dateutil
+, requests, tox, unicorn, intervaltree }:
 
 buildPythonPackage rec {
   version = "3.11.0";
```

Comment:
Please get rid of the `name` in the `pwntools` derivation.

Reply:
Ok, I thought it was mandatory, apparently not :P